tests/yast2_gui/yast2_control_center.pm

```diff
@@ -33,85 +33,110 @@ sub run() {
         send_key 'backspace';
     }
 
+
+    #	increase timeout value to 60 seconds for assert_screen and check_screen where is required
     #	start Add-on Products
     assert_and_click "yast2_control-center_add-on";
-    assert_screen "yast2_control-center_add-on_installed";
-    send_key "alt-c";
+    if (assert_screen "yast2_control-center_add-on_installed") {
```

Comment:
that does not make sense. assert_screen never returns with false, it dies the test